Firebase/InAppMessaging/Analytics/FIRIAMClearcutUploader.m

```diff
@@ -105,9 +105,6 @@ - (instancetype)initWithRequestSender:(FIRIAMClearcutHttpRequestSender *)request
     _nextValidSendTimeInMills = (int64_t)
         [_userDefaults doubleForKey:FIRIAM_UserDefaultsKeyForNextValidClearcutUploadTimeInMills];
 
-    // seed the first send upon SDK start-up
-    [self scheduleNextSend];
```

Comment:
Will want to hear back from @christibbs about this one. We may want to do this in case there are actually pending logs that are written to disk?

Reply:
Trying out an approach where we only call this if there are pending logs to be sent.